transcendence-app/src/socket/socket.gateway.ts

```diff
@@ -77,4 +83,19 @@ export class SocketGateway
   handleGetFriends(@ConnectedSocket() client: Socket) {
     this.socketService.getFriends(client);
   }
+
+  @SubscribeMessage('toggleUserWithRoles')
```

Comment:
Since I'm implementing this via ws, we don't really need the http endpoint to ass and remove usertorole anymore, but if I remove them, I also lose in the frontend the UserToRoleDto type, which I need. My solution has been to just leave those endpoints, as test endpoints, that enable swagger to document the UserToRoleDto class, and openapi generator create the correspoding type in webapp.
If someone has a different solution, I'd be interested in knowing that.